bot/exts/moderation/infraction/management.py

```diff
@@ -44,9 +45,25 @@ def infractions_cog(self) -> Infractions:
     # region: Edit infraction commands
 
     @commands.group(name='infraction', aliases=('infr', 'infractions', 'inf', 'i'), invoke_without_command=True)
-    async def infraction_group(self, ctx: Context) -> None:
-        """Infraction manipulation commands."""
-        await ctx.send_help(ctx.command)
+    async def infraction_group(self, ctx: Context, infr_id: int = None) -> None:
+        """Infraction manipulation commands. If `infr_id` is passed then this command fetches that infraction."""
+        if not infr_id:
```

Comment:
A minor nitpick: this will enter the condition with infr_id == 0. It's not actually important, but more correct to check explicitly for None.

Reply:
True. An infraction id will never be 0 since the count starts at 1, but will make this change nonetheless :+1:

Reply:
Done :+1: